util/stmtsummary/statement_summary.go

```diff
@@ -884,3 +885,19 @@ func convertEmptyToNil(str string) interface{} {
 	}
 	return str
 }
+
+// NeedRecord judges whether this type of statement should be summarized.
+func NeedRecord(stmtNode ast.StmtNode) bool {
+	switch stmtNode.(type) {
+	case ast.DMLNode, ast.DDLNode:
+		return true
+	case *ast.BeginStmt, *ast.DeallocateStmt, *ast.FlushStmt, *ast.SetConfigStmt, *ast.SetDefaultRoleStmt,
```

Comment:
commit stmt?

Reply:
`commit` is important, because in optimistic transactions, `commit` is always slow.